google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -2672,6 +2674,22 @@ Blob createFrom(
    */
   ReadChannel reader(BlobId blob, BlobSourceOption... options);
 
+  /**
```

Comment:
I wasn't sure if the code sample was required for each of these javadoc comments? I can add it if wanted.

Reply:
Yeah, let's add the small sample here similar to the other methods.